app.py

```diff
@@ -58,7 +59,7 @@ def indexPage():
 
 ## view details of all the patients present in the "patients" table
 @app.route('/patients')
-def viewPatientDetails():
+def viewAllPatientDetails():
```

Comment:
changing this name can affect the other parts of the code [ such as url_for( ).. and HTML templates ] so **do not** change this